spec/java_buildpack/util/file/droplet_sandbox_path.rb

```diff
@@ -0,0 +1,21 @@
+# Encoding: utf-8
```

Comment:
This file shouldn't need to exist.  The droplet sandbox can always be accessed (within as test) as `sandbox`.  So you can use straight `.to eq(...)` for expectations.